src/languages/typescript.js

```diff
@@ -53,15 +56,16 @@ export default function(hljs) {
   const TS_SPECIFIC_KEYWORDS = [
     "type",
     "namespace",
-    "typedef",
     "interface",
     "public",
     "private",
     "protected",
     "implements",
     "declare",
     "abstract",
-    "readonly"
+    "readonly",
+    "enum",
+    "keyof"
```

Comment:
`keyof` is not a keyword, it's a contextual keyword, so I'm not sure we should add it.

https://github.com/highlightjs/highlight.js/issues/3403